Crash when passing appengine-url instead of astarte-url

When this command it's executed
```
astartectl --appengine-url "https://api.your.astarte.com/appengine" -k test_private.pem -r test appengine devices data-snapshot
```
It crashes with
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7c9c35]

goroutine 1 [running]:
github.com/astarte-platform/astartectl/client.(*RealmManagementService).GetInterface(0x0, 0x7fff81f2cff6, 0x3, 0xc0005dc150, 0x25, 0x1, 0xc00012a000, 0x319, 0x0, 0x0, ...)
        /home/runner/work/astartectl/astartectl/client/realm_management.go:71 +0x65
github.com/astarte-platform/astartectl/cmd/appengine.devicesGetSamplesF(0x2200040, 0xc000158ab0, 0x3, 0x9, 0x0, 0x0)
        /home/runner/work/astartectl/astartectl/cmd/appengine/device.go:560 +0x1b66
github.com/spf13/cobra.(*Command).execute(0x2200040, 0xc00030f380, 0x9, 0xc, 0x2200040, 0xc00030f380)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0x21fe9c0, 0x43d54a, 0x219e8a0, 0xc000000180)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/astarte-platform/astartectl/cmd.Execute()
        /home/runner/work/astartectl/astartectl/cmd/root.go:51 +0x31
main.main()
        /home/runner/work/astartectl/astartectl/main.go:20 +0x20
```

`astartectl` should, instead, check that also `--realm-management-url` is present (or omit Realm Management checks) when `--appengine-url` is passed.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [0.11.0-rc.1] - Unreleased
 ### Fixed
 - appengine: Fix crash when retrieving nil values out of device interfaces
+- appengine: Fix panic when passing appengine-url without realmmanagement-url (#73)
 
 ### Changed
 - Moved the codebase to use astarte-go instead of the internal replicated tree

--- a/cmd/appengine/device.go
+++ b/cmd/appengine/device.go
@@ -250,15 +250,16 @@ func devicesDataSnapshotF(command *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	skipRealmManagementChecks = skipRealmManagementChecks || astarteAPIClient.RealmManagement == nil
 	if skipRealmManagementChecks && snapshotInterface == "" {
-		return fmt.Errorf("When using --skip-realm-management-checks, an interface should always be specified")
+		return fmt.Errorf("When not using Realm Management checks, an interface should always be specified")
 	}
 	interfaceTypeString, err := command.Flags().GetString("interface-type")
 	if err != nil {
 		return err
 	}
 	if skipRealmManagementChecks && interfaceTypeString == "" {
-		return fmt.Errorf("When using --skip-realm-management-checks, --interface-type should always be specified")
+		return fmt.Errorf("When not using Realm Management checks, --interface-type should always be specified")
 	}
 
 	outputType, err := command.Flags().GetString("output")
@@ -537,6 +538,7 @@ func devicesGetSamplesF(command *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	skipRealmManagementChecks = skipRealmManagementChecks || astarteAPIClient.RealmManagement == nil
 	forceAggregate, err := command.Flags().GetBool("aggregate")
 	if err != nil {
 		return err

--- a/utils/client.go
+++ b/utils/client.go
@@ -33,7 +33,7 @@ func APICommandSetup(individualURLVariables map[misc.AstarteService]string, keyV
 	for k, v := range individualURLVariables {
 		urlOverride := viper.GetString(v)
 		if urlOverride != "" {
-			individualURLVariables[k] = urlOverride
+			individualURLs[k] = urlOverride
 		}
 	}
 

